pyproject.toml

```diff
@@ -32,6 +32,7 @@ dependencies = [
     "flash-attn>=2.8.0.post2; platform_system != 'Darwin'",
     "liger-kernel>=0.5.4; platform_system != 'Darwin'",
     "flashinfer-python==0.2.8; platform_system != 'Darwin'",
+    "gsutil",
```

Comment:
One thing I found testing this is that this means that when we run experiments on augusta, the automated dataset upload tries to use the gsutil installed by uv (`/weka/oe-adapt-default/hamishi/pr_review/open-instruct/.venv/bin/gsutil`), but in a dev image, calling gsutil calls the image's version (`/usr/bin/gsutil`), which is different, and actually the auth from running `gcloud auth login` doesn't seem to transfer to the uv-installed gsutil.

Maybe we just remove this and directly install gsutil in the docker image?

Reply:
ah annoying. google tools be mysterious.